src/components/TextEditor.vue

```diff
@@ -127,6 +131,9 @@ export default {
 					priority: 'highest',
 				}
 			)
+			if (this.focus) {
+				logger.debug('focusing TextEditor')
```

Comment:
This should probably be removed? :)

Reply:
I'm fine with leaving those as they aid debugging of bug reports.